src/core/annotation.js

```diff
@@ -200,7 +200,10 @@ function getQuadPoints(dict, rect) {
 
       // The quadpoints should be ignored if any coordinate in the array
       // lies outside the region specified by the rectangle.
```

Comment:
Let's extend this comment to mention why `rect` can be `null`:

```
      // The quadpoints should be ignored if any coordinate in the array
      // lies outside the region specified by the rectangle. The rectangle
      // can be `null` for markup annotations since their rectangle may be
      // incorrect (fixes bug 1538111).
````